bar-trender-front-end/src/views/FilterResults.js

```diff
@@ -21,14 +21,38 @@ function ListPage() {
   });
 
   //  this.state = {establisments:{}}
-
+  const queryString = window.location.search;
+  const urlParams = new URLSearchParams(queryString);
+  console.log(urlParams.get('zones'));
+  let zones = urlParams.get('zones')
+  let beers = urlParams.get('beers')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/BarTrender/BarTrenderFront/pullRequest?prid=7143775)
```suggestion
  let beers = urlParams.get("beers")
```